.github/workflows/ci.yml

```
@@ -45,4 +110,264 @@ jobs:
         uses: codecov/codecov-action@v2
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          file: coverage.txt
+          flags: server
+
+  slack-notification:
+    name: Slack Notification
+    needs: [ci-web, ci-server]
+    if: success() || failure()
+    runs-on: ubuntu-latest
+    steps:
+      - name: Slack Notification
+        uses: Gamesight/slack-workflow-status@master
+        if: always()
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  info:
+    name: Collect information
+    needs: [ci-web, ci-server]
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' && (contains(needs.ci-server.result, 'success') || contains(needs.ci-web.result, 'success')) && github.event.repository.full_name == 'reearth/reearth' && (github.event.workflow_run.head_branch == 'release' || github.event.workflow_run.head_branch == 'main' || !startsWith(github.event.head_commit.message, 'v')) }}
```

```suggestion
    if: ${{ github.event_name == 'push' && (contains(needs.ci-server.result, 'success') || contains(needs.ci-web.result, 'success')) && github.event.repository.full_name == 'reearth/reearth' && (startsWith(github.event.workflow_run.head_branch, 'release/') || github.event.workflow_run.head_branch == 'main' || !startsWith(github.event.head_commit.message, 'v')) }}
```
Should we check `release/` prefix?

---

Been thinking about this too, I guess we'll have decide on how we are do the releases.

---

So isn't this line needed?
https://github.com/reearth/reearth/pull/393/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR5

---

That's what I'm talking about, like we need decide on how the release branches should look like.
i.e `release-0.0.1` is an option `release/0.0.1` is an option `release_001` is also an option.

---

I think you should keep same release strategy for now.
Also if we need to change some strategy, you can suggest it in other PR.
I think including a lot of change in same PR will be confused when we want to know why this change is needed.

---

Or is it difficult to keep same release strategy?

---

> Or is it difficult to keep same release strategy?

It'll do for now, ig.

ref: https://github.com/reearth/reearth/pull/393/commits/5ead45c127cd465ed575b890161bd226f4ee396f